Packages/Ethereum/.swiftpm/xcode/xcshareddata/xcschemes/potato.xcscheme

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why "potato"? What is the purpose of this scheme?

Reply:
the potato scheme builds the potato tool which is a generator of the safe ABIs (for now)